apps/nextjs-website/src/components/organisms/GitBookContent/components/Swagger/hooks/use-spec.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable functional/no-expression-statements */
```

Comment:
I think you should move this comment to line 10

Reply:
If I move this comment to line 10, I get the same eslint error for subsequent lines. 
I would to prefer to disable this rule for the entire file instead of adding a single comment above each error line